cpp/models/lct_secir/README.md

```diff
@@ -1,23 +1,59 @@
 # LCT SECIR model
 
 This model is based on the Linear Chain Trick. 
+
+The Linear Chain Trick provides the Option to use Erlang-distributed sojourn times in the compartments through the use of subcompartments. 
+The normal ODE models have (possibly unrealistic) exponentially distributed sojourn times.
+The LCT model can still be described by an ordinary differential equation system.
+
 For the concept see 
 - Lena Plötzke, "Der Linear Chain Trick in der epidemiologischen Modellierung als Kompromiss zwischen gewöhnlichen und Integro-Differentialgleichungen", 2023. (https://elib.dlr.de/200381/, German only)
 - P. J. Hurtado und A. S. Kirosingh, "Generalizations of the ‘Linear Chain Trick’: incorporating more flexible dwell time distributions into mean field ODE models“, 2019. (https://doi.org/10.1007/s00285-019-01412-w)
 
 The eight compartments 
-- Susceptible, may become exposed at any time
-- Exposed, becomes infected after some time
-- InfectedNoSymptoms, becomes InfectedSymptoms or Recovered after some time
-- InfectedSymptoms, becomes InfectedSevere or Recovered after some time
-- InfectedSevere, becomes InfectedCritical or Recovered after some time
-- InfectedCritical, becomes Recovered or Dead after some time
-- Recovered
-- Dead
+- `Susceptible` (S), may become exposed at any time
+- `Exposed` (E), becomes infected after some time
+- `InfectedNoSymptoms` (C), becomes InfectedSymptoms or Recovered after some time
+- `InfectedSymptoms` (I), becomes InfectedSevere or Recovered after some time
+- `InfectedSevere` (H), becomes InfectedCritical or Recovered after some time
+- `InfectedCritical` (U), becomes Recovered or Dead after some time
+- `Recovered` (R)
+- `Dead` (D)
 
 are used to simulate the spread of the disease. 
 It is possible to include subcompartments for the five compartments Exposed, InfectedNoSymptoms, InfectedSymptoms, InfectedSevere and InfectedCritical.
 
-A simple example can be found at: examples/lct_secir.cpp. 
+Below is an overview of the model architecture and its compartments.
+
+![tikzLCTSECIR](https://github.com/SciCompMod/memilio/assets/70579874/e385f26c-5070-4655-9faf-eba753dc8e99)
```

Comment:
Very nice overview ;) However, we should remove the describtions C, I ... and name them `I_NS` , `I_SY`  as in the other models.

Reply:
I have not used the variables so far because I would have so many indices (e.g. I_NS_n_I_NS for the last subcompartment of I_NS ... ). Do you have another suggestion for this?

Reply:
Talked to @mknaranja about this. Naming should be okay now.